azure-pipelines.yml

```diff
@@ -18,6 +18,46 @@ variables:
 
 
 jobs:
+- job: test_aswfdocker
+  displayName: Test aswfdocker
+  timeoutInMinutes: 0
+  pool:
+    vmImage: 'ubuntu-18.04'
+  steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.7'
+
+    - bash: pip3 install pipenv
+      displayName: Install pipenv
+
+    - bash: pipenv install --dev
+      displayName: Install aswfdocker
+
+    - bash: pipenv run pytest python/aswfdocker --doctest-modules --junitxml=test-pytest-results.xml --cov=. --cov-report=xml --cov-report=html
```

Comment:
Might be useful to add a comment that the XML output format of pytest-cov is in fact the Cobertura format, which is why you can then use the PublishCodeCoverageResults@1 task to consume these results. It doesn't seem like this is a very well documented fact (I only found indirect references to that after a bit of googling), and the Cobertura tool itself is Java specific, so that's confusing in the context of Python.